.gitignore

```diff
@@ -7,4 +7,5 @@ venv/
 .pytest_cache/
 **__pycache__/
 *.pyc
-app/db.sqlite3
\ No newline at end of file
+db.sqlite3
+cinema_service_db_data.json
```

Comment:
cinema_service_db_data.json should not be included in gitignore and should not be committed